packages/nextjs/utils/scaffold-eth/networks.ts

```diff
@@ -37,9 +152,13 @@ export const NETWORKS_EXTRA_DATA: Record<string, ChainAttributes> = {
   },
   [chains.optimismGoerli.id]: {
     color: "#f01a37",
+    etherscanEndpoint: "https://api-goerli-optimism.etherscan.io",
+    etherscanApiKey: MAIN_ETHERSCAN_API_KEY,
   },
   [chains.optimism.id]: {
     color: "#f01a37",
+    etherscanEndpoint: "https://api-goerli-optimism.etherscan.io",
+    etherscanApiKey: MAIN_ETHERSCAN_API_KEY,
```

Comment:
Probably a copy&paste typo here, I think it should be: `https://api-optimistic.etherscan.io`

I'm also having problems using `MAIN_ETHERSCAN_API_KEY` for OP Mainnet, I had to create an Optimistic Etherscan account & API to make it work. 

Should we create a `OPTIMISTIC_ETHERSCAN_API_KEY` with a shared API for development?

Reply:
Ohh yes, tysm for the catch !! updated them also added some new networks 

also updated `.env.example`,  I can share my `.env.local` on TG DM for testing, also updated the main comment with TODO to add those in vercel environment 🙌